task/04-arrays-tasks.js

```diff
@@ -454,7 +536,16 @@ function getIdentityMatrix(n) {
  *     3, 3   => [ 3 ]
  */
 function getIntervalArray(start, end) {
-   throw new Error('Not implemented');
+   let newArr = [];
+   function createArrayOfNumbers(){
+      if (end>=start){
```

Comment:
it is necessary to observe indents.
Example: "if (a > b) {"
https://javascript.info/coding-style